src/cryptonote_basic/cryptonote_basic_impl.cpp

```diff
@@ -376,10 +376,8 @@ bool get_account_address_from_str_or_url(
 {
 	if(get_account_address_from_str(nettype, info, str_or_url))
```

Comment:
I would remove that function completely and replace the references with `get_account_address_from_str` as the name now is misleading.